maze_transformer/training/tokenizer.py

```diff
@@ -60,3 +71,92 @@ def as_tokens(
             )
 
         return tokens
+
+
+class HuggingMazeTokenizer(PreTrainedTokenizer):
+    vocab: dict[str, int]  # map of token_ids to strings
+
+    bos_token: str = SPECIAL_TOKENS["start_path"]
```

Comment:
I renamed these to `path_start` and `path_end` in the other PR to be consistent with the other special tokens, plz forgive me